theano/gpuarray/dnn.py

```diff
@@ -1700,24 +1707,12 @@ def __setstate__(self, d):
             self.inplace_output = False
             self.destroy_map = {}
 
-    def get_op_params(self):
-        params = []
-        if self.inplace_output:
-            params.append(('INPLACE_OUTPUT', '1'))
-        if self.running_averages:
-            params.append(('RUNNING_AVERAGES', '1'))
-            if self.inplace_running_mean:
-                params.append(('INPLACE_RUNNING_MEAN', '1'))
-            if self.inplace_running_var:
-                params.append(('INPLACE_RUNNING_VAR', '1'))
-        params.append(('MODE', ("CUDNN_BATCHNORM_SPATIAL"
-                                if self.mode == "spatial"
-                                else "CUDNN_BATCHNORM_PER_ACTIVATION")))
-        return params
-
     def infer_shape(self, node, shape):
         return [shape[0]] + [shape[1]] * (len(node.outputs) - 1)
 
+    _cop_num_inputs = 7
```

Comment:
it would be better to have the attribute at the top of the class, next to props.